modules/http/http.c

```diff
@@ -466,6 +496,8 @@ _flush(LogThreadedDestDriver *s)
   if (self->super.batch_size == 0)
     return WORKER_INSERT_RESULT_SUCCESS;
 
+  _finish_request_body(self);
```

Comment:
Sorry if my comment is a matter-of-taste comment (i.e. how I would organize the code),
but I think prefix and suffix should be part of message construction and not part of flush().
Flush should be about the actual message sending and the logic around it, in my opinion.

e.g. _insert_batched()
``` diff
if (_should_initiate_flush(self))
    {
+     _finish_request_body(self);
      return _flush(&self->super);
    }
```
insert_single()
``` diff
self->request_headers = _format_request_headers(self, msg);
+init_request_body(self);
_add_message_to_batch(self, msg);
+_finish_request_body(self);
return _flush(&self->super);

```
What do you think?